content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt

```diff
@@ -0,0 +1,57 @@
+Title: The order of kirbytext/kirbytags hooks
```

Comment:
The terms are used in mixed casing throughout the PR. According to our content styleguide, it should be `KirbyText` and `KirbyTags`.

Reply:
Hm, the hooks itself are called `kirbytext:before` etc. in the docs, without camelCase, so I wasn't sure what to use where.

Reply:
The hooks yes, but otherwise I‘d use the uppercase forms.